### ISSUE TITLE
Problem with getting metrics on test and validation sets.

### ISSUE THREAD

**Issue by vadimdddd** [open-mmlab/mmdetection#11920]
Hello, I have some problem with training process. I have tasks: 1) plot graph train and val loss to epochs; 2) plot graph train and val mAP and AP by classes to epochs; also 3) have to add precision in output table. I have the following versions: mmdet = 2.27.0. I tried to plot train/val loss using workflow: [('train', 1), ('val', 1)], got some error, also tried to figure out hooks(after_train_epoch, after_val_epoch, TensorboardLoggerHook, but my experiments have been unsuccessful so far. Can smb help me please? My cfg + error later: 
[asda.txt](https://github.com/user-attachments/files/16710026/asda.txt)
